package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
+  },
```

Comment:
- [ ] Nice work configuring Webpack command for production mode (`"build": "webpack"`) :clap: However the command for building Webpack in development mode is not configured (`"start": "webpack serve --open"`). Kindly make sure it's configured and the command `npm run start` or `npm run dev` run the **dev server**.